pom.xml

```diff
@@ -138,6 +139,18 @@
     </developer>
   </developers>
 
+  <distributionManagement>
+    <repository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+    </repository>
+    <snapshotRepository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+  </distributionManagement>
+
+  <!--
   <distributionManagement>
```

Comment:
IIRC, this section is required

Reply:
Yes, I've replaced it.  I need to remove the commented section.  That contains the JCenter settings.